src/drive/web/modules/drive/FileOpenerExternal.jsx

```diff
@@ -27,16 +27,27 @@ class FileOpener extends Component {
     file: null
   }
   componentWillMount() {
-    this.loadFileInfo()
+    if (this.props.fileId) {
+      this.loadFileInfo(this.props.fileId)
+    } else {
+      if (this.props.routeParams.fileId) {
```

Comment:
Out of curiosity, why use `else { if {` instead of an `else if` statement? I find nested `if`'s to be a code smell, but that might just be me.

Reply:
Because `else if` doesn't exist in JS and I don't like to rely on indentation ? :D 

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/if...else#Description

Reply:
Well we're using blacks with `{ }` here so it wouldn't rely on indentation :)